src/components/v2/shared/FundingCycleConfigurationDrawers/__tests__/FundingCycleDrawer.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+
+import { useFundingCycleDrawer } from '../useFundingCycleDrawer'
+
+describe('FundingCycleDrawer', () => {
+  describe('useFundingCycleDrawer', () => {
+    const instanceUnderTest = (onClose: VoidFunction = () => {}) => {
+      const { result } = renderHook(() => useFundingCycleDrawer(onClose))
+      return result
+    }
+    test('calls onClose when emitDrawerClose is called', () => {
+      let called = false
+      const result = instanceUnderTest(() => (called = true))
+      result.current.emitDrawerClose()
+      expect(called).toEqual(true)
```

Comment:
an alternative might be (not sure if it works lol):

```suggestion
      const callback = jest.fn()
      const result = instanceUnderTest(callback)
      result.current.emitDrawerClose()
      expect(callback).toHaveBeenCalled()
```

Reply:
TIFL 🙌